code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -43,16 +43,6 @@
 	src.add_fingerprint(usr)
 	return
 
-/obj/item/weapon/card/data/clown

```

Comment:
I asked in coderbus about removing it, and was told that it wasn't used anywhere and should be removed.